### PR TITLE
Add TravisCI support. Implements #42

### PR DESCRIPTION
**Pull request by JBaczuk**
Implements #42

### CODE REVIEW

File: .btcdeb_history [hunk 1]
```diff
@@ -0,0 +1,4 @@
+setp
```
Comment (wardlem) [inline .btcdeb_history]: What is this file?  Is it required by Travis?

File: test.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+const bs = require('./index.js')
```
Comment (wardlem) [inline test.js]: Does this file need to be here?

File: .btcdeb_history (hunk 1 above)
Comment (JBaczuk) [reply .btcdeb_history]: oh no this is from running `btcdeb` in this folder. I should remove it.

File: test.js (hunk 2 above)
Comment (JBaczuk) [reply test.js]: no

**Comment by JBaczuk** [comment]
@wardlem cleaned up, does it look better now?

**Pull request merged**